analyzers/tests/SonarAnalyzer.Test/TestCases/ExceptionsShouldBeLogged.Microsoft.Extensions.Logging.cs

```diff
@@ -0,0 +1,405 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+// Logging methods from: https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerextensions?view=dotnet-plat-ext-8.0
+public class TestCases
+{
+    private readonly ILogger logger = new Logger<TestCases>(new NullLoggerFactory());
+
+    public void Simple()
+    {
+        try { }
+        catch (Exception e)
+        {
+            logger.Log(LogLevel.Critical, e, "Message!");
+        }
+
+        try { }
+        catch (Exception e)
+        {
+            logger.Log(LogLevel.Critical, "Message!"); // Noncompliant
```

Comment:
Add at least one test case that shows the location where the rule raises the issue (`^^^^^^^^`).